packages/app/src/cli/services/dev/extension/bundler.test.ts

```diff
@@ -554,6 +556,44 @@ describe('setupFunctionWatcher', () => {
     expect(chokidarOnSpy).toHaveBeenCalledWith('change', expect.any(Function))
   })
 
+  test('deploys the function on file change in locales directory', async () => {
+    // GIVEN
+    // fileExists returns true
```

Comment:
Nit: In other tests it looks like this is formatted differently. I also don't think we need the extra comment, it's easy to tell by looking at the code 😄 

```suggestion
    // Given
```